arangod/Aql/SortingGatherExecutor.cpp

```diff
@@ -358,14 +358,14 @@ auto SortingGatherExecutor::produceRows(typename Fetcher::DataRange& input,
     // First initialize
     auto const callSet = initialize(input, output.getClientCall());
     if (!callSet.empty()) {
-      return {ExecutorState::HASMORE, NoStats{}, callSet};
+      return {ExecutorState::HASMORE, NoStats{}, std::move(callSet)};
```

Comment:
Move const value is copy

Same in other places here

Reply:
Fixed